README.md

```diff
@@ -67,6 +69,8 @@ We would love contributes that fall under the 'Planned Features' and fixing any
 
 ### Using
 
+This library needs at least Go 1.13.
```

Comment:
Go modules should be enabled.

Reply:
That's the case by default on Go 1.13, so that doesn't seem like something we need to explicitly call out.

Reply:
https://golang.org/doc/go1.13#modules

> The GO111MODULE environment variable continues to default to auto, but the auto setting now activates the module-aware mode of the go command whenever the current working directory contains, or is below a directory containing, a go.mod file — even if the current directory is within GOPATH/src. This change simplifies the migration of existing code within GOPATH/src and the ongoing main

Reply:
Go 1.13 default is `auto`, so it depends on the directory structure.
(I recently asked by some people about build problem due to this.)

Reply:
Sure, but that'll only go wrong if you're using 1.13 and you don't have a `go.mod` in the package you're building. Which on Go 1.13 means you're kinda doing it wrong. Things should have a go.mod by now, which will activate module mode by default.

Reply:
I think something like `Go modules should be enabled.` and link to https://github.com/golang/go/wiki/Modules is enough.

Reply:
I think that's overdoing it, but fine. Made the change.